### DISCUSSION TITLE
Aplicação full stack

### DISCUSSION THREAD

**Discussion by Matheusmauriciom** [balta-io/forum#505] [Tira Dúvidas] [answered]
Boa tarde mestre Balta!!

Estou desenvolvendo uma aplicação utilizando o stack Microsoft (C#, ASP.NET, etc...).

Inicialmente, criei um banco de dados e, em seguida, estou desenvolvendo uma API (ASP.NET Core) para interagir com esse banco, e irei utilizar o Swagger para testes e documentação. Em seguida minha ideia é criar um projeto Blazor (WebAssembly/ Server) para funcionar como o frontend da aplicação.

Minha principal dúvida está relacionada à integração entre a API e o projeto Blazor. Suponha-se que configurei a rota da API como [Route("api/questions")], enquanto no Blazor, estou utilizando @page "/questions" para a página que exibirá as perguntas. Essa abordagem parece "coerente/valida" para estabelecer a comunicação entre ambas as partes?  (lembrando que é apenas um exemplo) 

Confesso que não sei se já explicou essa parte em alguma carreira/aula ou até mesmo no youtube, e gostaria de saber em qual carreira eu posso me aprofundar para entender esse processo de comunicação entre o consumo de api para meu front.
Obrigado <3

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @Matheusmauriciom, como vai?

Sim, é a abordagem que usamos nos cursos inclusive! Gosto de usar o prefixo "api"!
